lib/beaker-pe/install/pe_utils.rb

```diff
@@ -289,7 +289,11 @@ def fetch_pe(hosts, opts)
         # @param [Host] host The host to install pacakges for
         # @api private
         def deploy_frictionless_to_master(host)
-          klass = host['platform'].gsub(/-/, '_').gsub(/\./,'')
+          platform = host['platform']
+          if platform == "aix-7.2-power"
```

Comment:
Yeah, I can add a comment to that effect. I will definitely be in here messing with this again in the agent 2.0 timeframe.